subjects/localhost/README.md

```diff
@@ -0,0 +1,162 @@
+## Localhost
+
+The Hypertext Transfer Protocol (HTTP) is an application layer protocol in the Internet protocol suite model for distributed, collaborative, hypermedia information systems.
+HTTP is the foundation of data communication for the World Wide Web. Hypertext documents include hyperlinks to other resources that the user can easily access, for example, by a mouse click or tap on the screen in a web browser.
+
+### Instructions
+
+#### Server
+
+The project can be written in one of these languages [Go,C++,C...]
+
+```console
+$ ./localhost [CONFIG FILE]
+Lisning on  [IP]:[PORT]
+Lisning on  [IP]:[PORT2]
+```
+- Your server must not crash for any reason.
+- Any request to your server should never hang forever.
+- Your server can listen on multiple servers and ports at the same time without any conflict 
+- You must use the single process and no treads.
+- You must use `exec` functions only in `CGI` part
```

Comment:
`exec` is very language specific. To be more language agnostic speaking about "forking a process" instead, which is a common terminology for all languages supporting those features.

Reply:
The exec  function families exist in Rust and Go C++ and C:
 exec in C: they are defined in the header unistd.h
exec in go: os/exec
exec in rust : std::process::Command
exec in c++ : exec